Minio.Examples/Cases/ListenNotifications.cs

```diff
@@ -0,0 +1,51 @@
+/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2024 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Minio.DataModel.Args;
+using Minio.DataModel.Notification;
+
+namespace Minio.Examples.Cases;
+
+internal static class ListenNotifications
+{
+    // Listen for gloabal notifications (a Minio-only extension)
+    public static void Run(IMinioClient minio,
+        List<EventType> events = null)
+    {
+        IDisposable subscription = null;
+        try
+        {
+            Console.WriteLine("Running example for API: ListenNotifications");
+            Console.WriteLine();
+            events ??= new List<EventType> { EventType.BucketCreatedAll };
+            var args = new ListenBucketNotificationsArgs().WithEvents(events);
+            var observable = minio.ListenNotificationsAsync(events);
+
+            subscription = observable.Subscribe(
+                notification => Console.WriteLine($"Notification: {notification.Json}"),
+                ex => Console.WriteLine($"OnError: {ex}"),
+                () => Console.WriteLine("Stopped listening for bucket notifications\n"));
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine($"[Bucket]  Exception: {e}");
+        }
+        finally
+        {
+            subscription?.Dispose();
+        }
+    }
```

Comment:
What's the purpose of this method? It will start listening, listen for events via a subscription and terminate that subscription right away.

Reply:
Thanks for the feedback! Would keeping the subscription alive for a few minutes and then disposing be a better approach?

Reply:
If it's an example, then I would implement something that raises a notification. Once it has been received, then it should show it and then return. I'm working (with very low prio) on a complete new MinIO .NET SDK and the integration tests does it like this: https://github.com/ramondeklein/minio-dotnet2/blob/main/Minio.IntegrationTests/Tests/BucketNotificationTests.cs.

Reply:
If you want to do it more quick'n'dirty, then you can enter `Console.ReadLine()` and let the user decide when to stop listening...

Reply:
@adelinag08 Should be public now, but it needs some more work...